test/ex_l7/ack/sequence_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule ExL7.Ack.SequenceTest do
+  use ExUnit.Case
+  doctest ExL7.Ack.Sequence
+
+  import ExL7.Ack.Sequence
+
+  test "two sequences should not be the same" do
+    {:ok, sequencer} = start_link()
+    assert get_next(sequencer) != get_next(sequencer)
+  end
+
+  test "sequence counter should increase" do
+    {:ok, sequencer} = start_link()
+    num1 = String.slice(get_next(sequencer), -1..-1)
+    num2 = String.slice(get_next(sequencer), -1..-1)
+
+    assert num1 != num2
```

Comment:
To be accurate with the test, should this assert that `num2` is greater than `num1` rather than checking that they aren't the same?

Reply:
They are a string mix of a date time string + a number, it's not possible. I changed the variables to be called sequence1 and sequence2 for clarity, though.